server/src/main/java/com/distkv/server/Service.java

```diff
@@ -0,0 +1,48 @@
+package com.distkv.server;
+
+import java.io.Closeable;
+
+/**
+ * Service LifeCycle.
+ *
+ */
+public interface Service extends Closeable {
+
+  enum ServiceStatus {
```

Comment:
Is this double defined? I saw another definition in ServiceStatus.java

Reply:
I will check this.